docs/datagrid/your-first-datagrid.md

```diff
@@ -1,7 +1,7 @@
 Your first DataGrid
 ===
 
-When you are ready to create your first simple DataGrid, please follow these steps:
+Follow these steps to create your first simple DataGrid:
 
 ### 1. Add a div to your webpage with a specific id.
```

Comment:
```suggestion
### 1. Add a div
Add a `div` to your webpage with a specific id.
```

Reply:
Didn't manage to add inline comment, but "###2. In this basic...." below should be changed to:

### 2. Import (or install) the library
In this basic example we are going to create a simple DataGrid. First we have to import (or install) the DataGrid library.